src/datasource.js

```diff
@@ -23,6 +24,12 @@ CSSDataSource.prototype.getRamp = function (column, bins, method, callback) {
     ramp = d3.scale.quantile().range(d3.range(bins)).domain(values.map(function (f) {
       return f.properties[column]
     })).quantiles()
+  } else if (method === 'jenks') {
+    var valuesInGeoJSON = {
+      "type": "FeatureCollection",
+      "features": values
+    }
+    ramp = jenks(valuesInGeoJSON, column, bins);
   } else {
     throw new Error('Quantification method ' + method + ' is not supported')
```

Comment:
Function has a callback, but here we are throwing an error. This method is synchronous but, by default, we've defined datasources asynchronous. So, I don't know if we should implement this method without callback or with callback and return error instead of throwing it.